### DISCUSSION TITLE
Organization discussions tab

### DISCUSSION THREAD

**Discussion by daviddavis** [community/community#4031] [Discussions]
When I go to https://github.com/github, there is a tab there that shows Discussions across the github organization. I don't see a way to enable this for my organization? We'd really like to have this as it would help to make Discussions more discoverable. Thanks.

**Comment by froschdesign** [comment]
> …there is a tab there that shows Discussions across the github organization.

Or from others: https://github.com/orgs/github/discussions?discussions_q=org%3Apulp 😉 

> We'd really like to have this as it would help to make Discussions more discoverable.

It would be a central entry point and it could be shared with one link.

**Reply by quba42** [reply]
The single query for all the pulp discussions is definitely already helpful.

I guess the original question is if there is some way (organization setting?) to make the tab appear here: https://github.com/pulp? In the same way as it is present here: https://github.com/github.

**Reply by froschdesign** [reply]
> I guess the original question is if there is some way (organization setting?) to make the tab appear here: https://github.com/pulp? In the same way as it is present here: https://github.com/github.

Right, this is the wish. (The link from me was only a hint that there is an _unbeautiful_ workaround, which is not the goal of the request!)

**Comment by froschdesign** [comment]
Too bad, discussions [is out of beta](https://github.blog/2021-08-17-github-discussions-out-of-beta/) but the tab is still missing for organizations. :disappointed:

**Reply by volkantash** [reply]
It's a feature for them. Can't it be easily created for everyone? Engrossing.

**Comment by volkantash** [comment]
Hello? Did anyone developer see this? We are waiting...

**Comment by evi-liu** [comment]
👋 Hi folks, thanks for all of the helpful feedback on this. The Discussions org tab on the GitHub org is a just a little experiment we've been testing internally. We do have plans to build out a proper org-level Discussions experience. So stay tuned, and I'll update this thread once we have a private beta available for testing. Thank you for your patience.

**Reply by volkantash** [reply]
Thanks for information. We are waiting. ^^

**Comment by jayceeday** [comment]
Hello!

We have now launched Discussions for Organizations 🥳

You can read about [this launch in our blog.](https://github.blog/2022-04-12-whats-new-in-github-discussions-organization-discussions-polls-and-more/)
